src/MerchantCenter/MerchantStatuses.php

```diff
@@ -372,6 +370,13 @@ protected function refresh_account_issues(): void {
 			];
 		}
 
+		/**
+		 * Account-level issues can be added the merchant issues table.
```

Comment:
Missing `to`, or maybe it needs to be reworded to state that we can use this filter to add custom account issues which will be added to the table.
Is the same hook handy for product issues? Or will we add this when we need it?